Jellyfin.Api/Controllers/ClientLogController.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System.Net.Mime;
+using System.Threading.Tasks;
+using Jellyfin.Api.Attributes;
+using Jellyfin.Api.Constants;
+using Jellyfin.Api.Models.ClientLogDtos;
+using MediaBrowser.Controller.ClientEvent;
+using MediaBrowser.Controller.Configuration;
+using MediaBrowser.Controller.Net;
+using MediaBrowser.Model.ClientLog;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Jellyfin.Api.Controllers
+{
+    /// <summary>
+    /// Client log controller.
+    /// </summary>
+    [Authorize(Policy = Policies.DefaultAuthorization)]
+    public class ClientLogController : BaseJellyfinApiController
+    {
+        private const int MaxDocumentSize = 1_000_000;
+        private readonly IClientEventLogger _clientEventLogger;
+        private readonly IAuthorizationContext _authorizationContext;
+        private readonly IServerConfigurationManager _serverConfigurationManager;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ClientLogController"/> class.
+        /// </summary>
+        /// <param name="clientEventLogger">Instance of the <see cref="IClientEventLogger"/> interface.</param>
+        /// <param name="authorizationContext">Instance of the <see cref="IAuthorizationContext"/> interface.</param>
+        /// <param name="serverConfigurationManager">Instance of the <see cref="IServerConfigurationManager"/> interface.</param>
+        public ClientLogController(
+            IClientEventLogger clientEventLogger,
+            IAuthorizationContext authorizationContext,
+            IServerConfigurationManager serverConfigurationManager)
+        {
+            _clientEventLogger = clientEventLogger;
+            _authorizationContext = authorizationContext;
+            _serverConfigurationManager = serverConfigurationManager;
+        }
+
+        /// <summary>
+        /// Post event from client.
+        /// </summary>
+        /// <param name="clientLogEventDto">The client log dto.</param>
+        /// <response code="204">Event logged.</response>
+        /// <response code="403">Event logging disabled.</response>
+        /// <returns>Submission status.</returns>
+        [HttpPost]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        public async Task<ActionResult> LogEvent([FromBody] ClientLogEventDto clientLogEventDto)
+        {
+            if (!_serverConfigurationManager.Configuration.AllowClientLogUpload)
+            {
+                return Forbid();
+            }
+
+            var authorizationInfo = await _authorizationContext.GetAuthorizationInfo(Request)
```

Comment:
Technically, the information you want is already present in `HttpContext.User.Claims`

Reply:
Is this resolved though?